### ISSUE TITLE
Coded Login for the Mobile App (not functional)

### ISSUE THREAD

**Issue by catherineunciano** [IT3A-2016/Paper-Icebergs#5]

**Comment by giomigueltada** [comment]
<?php
	session_start();
	include_once ('db.php');
	
	$error=false;
	
	if(isset($_POST['login']))
	{
		if($_POST['user_name'] != '' && $_POST['password'] != '')
			{
				$name = mysqli_real_escape_string($connection, $_POST['user_name']);
				$pass = mysqli_real_escape_string($connection, $_POST['password']);
				$check_email = '';
				// Validate e-mail
				if (!filter_var($name, FILTER_VALIDATE_EMAIL) === false) {
					$check_email = $name;
				} 
				if($check_email!='')
				{	// email & password combination
					$query = "SELECT * FROM `user` WHERE `email` = '$name' AND `password` = '$pass'";
				} 
				else {	// username & password combination
					$query = "SELECT * FROM `user` WHERE `user_name` = '$name' AND `password` = '$pass'";
				}
				
				$result = mysqli_query($connection, $query);
					if($row=mysqli_fetch_array($result)){
						session_start();
						$_SESSION['login'] = "1";
						header("Location:dashboard.php");
					}
					else {
						echo "Invalid Login Credentials.";
					}
					
			}
	}
	 else{
	?>
<!DOCTYPE html>
<html>

<head>
	<title>QWS</title>
	
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1, description" content="">
	<meta name="author" content="">
	
	<link rel="stylesheet" href="../css/bootstrap.min.css" type="text/css">
	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
	
	<script src="../js/smoothscroll.js"></script>
	<script src="../js/webscript.js"></script>
	<script src="../js/jquery.js"></script>
	<script src="../js/bootstrap.min.js"></script>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
	<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
	
	<style>
	body {background-color:#2E86C1;}
	h1   {color: blue;}
	p    {color: red;}
	</style>
</head>
<body class="cusbodycolor" oncontextmenu="return false">
	<div class="row">
		<div class="col-sm-12">
			<div class="col-sm-12 well">
				<span style="font-family:'Times New Roman', Times, serif;font-size:20px;"/>Automated Queueing Web-based System
			</div>
		</div>
	</div>
	
	<div class="row" style="padding:20px 0 20px 0;">
		<br><br>
	</div>
	
	<div class="container">
		<div class="row">
			<div class="col-sm-12">
				<form role="form" action="<?php echo $_SERVER['PHP_SELF']; ?>" method="post" name="signupform">
					<fieldset>
					
						<div class="col-sm-3"></div>
						
						<div class="col-sm-6 well">
							<div class="row"><div class="col-sm-12 col-sm-offset-5">
								<h4><span style="font-family:'Times New Roman', Times, serif;font-size:30px;"/>Login<h4></div></div>
							
							<div class="col-sm-12">
								<div class="col-sm-12">
									<small>Email or Username</small>
									<input type="text" name="user_name" id="user_name" placeholder="Enter Email or Username" class="form-control borderless" style="margin-bottom:10px;" required />
								</div>
								
								<div class="col-sm-12">
									<small>Password</small>
									<input type="password" name="password" id="password" placeholder="Enter Password" class="form-control borderless" style="margin-bottom:10px;" required />
									<span class="text-danger"><?php if (isset($password_error)) echo $password_error; ?></span>
								</div>
							</div>
							
							<div class="col-sm-12"><hr></div>
							
							<div class="col-sm-4 col-sm-offset-10" style="align:right;">
								<input type="submit" name="login" value="Submit" class="btn btn-primary"/>
							</div>
						</div>
					</fieldset>
				</form>
			</div>	
		</div>
		
		<div class="row"><span class="note"><?php if (isset($msg)) { echo $msg; } ?></span></div>
		
		<div class="row">
			<div class="col-sm-4 col-sm-offset-4 text-center">	
			</div>
		</div>
	</div>
	
	<div class="row" style="padding:30px 0 30px 0;"><br><br></div>
	<footer style="text-align:center;background-color:#707B7C;color:#000000;">Paper-Icebergs 2017 &copy; All rights reserved.</footer>
	<?php } ?>
</body>
</html>
